testsuites/rio/src/main/java/org/eclipse/rdf4j/rio/AbstractParserHandlingTest.java

```diff
@@ -948,4 +951,15 @@ private final Model getTestModel(String languageValue, String languageTag) {
 				vf.createLiteral(languageValue, languageTag)));
 		return result;
 	}
+
+	@Test(expected = RDFParseException.class)
```

Comment:
Purpose of the test is to make parsing in presence of a null rdfHandler consistent. So I propose to give it input other than "" to force it to never be valid RDF across all parsers. Maybe something like "{(.;".